spec/support/macros.rb

```diff
@@ -0,0 +1,3 @@
+def set_current_user(user = nil)
```

Comment:
Unused method argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used. You can also write as `set_current_user(*)` if you want the method to accept any arguments but don't care about them.